wagtail/admin/viewsets/base.py

```diff
@@ -15,6 +18,9 @@ class ViewSet:
     #: A name for this viewset, used as the default URL prefix and namespace.
     name = None
 
+    #: The icon to use across the views.
+    icon = ""
```

Comment:
Not sure if this should be here because the base `ViewSet` doesn't have any preconfigured views, but I thought it'd be nice to standardise this here and so that the `menu_icon` uses this by default.

I extracted this into a separate commit just now, sorry for making further changes after sending the review request!